phpunit.xml

```diff
@@ -9,6 +9,7 @@
   <testsuites>
     <testsuite name="main">
       <directory>tests</directory>
+      <directory>rules-tests</directory>
```

Comment:
@JohJohan @TomasVotruba this somehow cause error when run test error when pull `rector-src:dev-main`, I possibly due to fixture conflict name, I will check ;)